docs/requirements.txt

```diff
@@ -7,5 +7,8 @@ matplotlib
 penzai
 scikit-learn
 
+# Needed to conver from Jupyter to markdown
+jupytext
```

Comment:
We shouldn't add this here, because it's not actually a documentation requirement – for example, we don't need readthedocs to install jupytext in order to build the documentation.

We could add a separate development requirements file, but that might be overkill because the only thing it would contain is jupytext. Maybe we just leave it for users to install directly? What do you think?

Reply:
Let's aim for simplicity and leave this out to be installed separately.